src/containers/my-quizzes/scroll-question/ScrollQuestions.tsx

```diff
@@ -0,0 +1,45 @@
+import { FC } from 'react'
+import { useTranslation } from 'react-i18next'
+import {
+  Box,
+  Typography,
+  Checkbox,
+  Radio,
+  FormGroup,
+  RadioGroup,
+  FormControlLabel
+} from '@mui/material'
+import { questionType } from '~/components/question-editor/QuestionEditor.constants'
+
+import { QuizQuestionProps } from '~/types'
+import { styles } from '~/containers/my-quizzes/scroll-question/ScrollQuestions.styles'
+
+const QuizQuestion: FC<QuizQuestionProps> = ({ question, index }) => {
```

Comment:
As I see this component is similar to SelectableQuestion, can you reuse it by passing different styles

Reply:
> As I see this component is similar to SelectableQuestion, can you reuse it by passing different styles?

It's make sense. 
I used this component for rendering and added conditional for render main wrapper container